src/player.cpp

```diff
@@ -6830,6 +6859,27 @@ std::list<item> player::use_charges( const itype_id& what, long qty )
         use_fire( qty );
         return res;
 
+    } else if( what == "breath" ) {
+        auto rebreathers = all_items_with_flag( "SCUBA" );
+        for( auto &i : rebreathers ) {
+            auto reb = charges_of( i->typeId(), ( long )ceil( qty * 0.5 ) );
+            if( reb > 0 ) {
+                auto found = use_charges( i->typeId(), reb );
+                res.splice( res.end(), found );
+                qty -= std::min( qty, long( reb / 0.5 ) );
```

Comment:
Instead of `/ 0.5` you could write that as `* 2`, which would avoid conversion to `double` and therefor also avoid the need to cast the result to `long`.